### ISSUE TITLE
Add range slider to the GUI

### ISSUE THREAD

**Issue by justinspencer** [WestpacCXTeam/GUI-source#435]
We've had renewed interest in the Range Slider component. We used to have one in the GUI however it was removed due to insufficient browser support and accessibility requirements. It looks like the accessibility issues have been resolved so we should probably try again.

Julie from intopia reckons this range slider implementation is pretty good (a11y wise) https://codepen.io/chriswrightdesign/pen/GgxVrL
